src/cart/crt_iv.c

```diff
@@ -2735,6 +2735,14 @@ handle_response_cb(const struct crt_cb_info *cb_info)
 	struct crt_rpc_priv	*rpc_priv;
 	struct crt_context	*crt_ctx;
 
+	/* handle locally generated errors during IV operations synchronously to ensure unregister
+	 * of bulk buffer will occur before freeing it, just in case peer will finally make it
+	 * unexpectedly
+	 */
+	if (cb_info->cci_rc == -DER_TIMEDOUT || cb_info->cci_rc == -DER_EXCLUDED ||
+	     cb_info->cci_rc == -DER_CANCELED)
```

Comment:
```suggestion
	    cb_info->cci_rc == -DER_CANCELED)
```

Reply:
> So perhaps something like (after this check): if (cb_info->cci_rc != DER_SUCCES) D_DEBUG(....)

Well, there is already the RPC_ERROR() in crt_rpc_complete() that should make it, do you agree ?

Reply:
The issue is that we might crash before we get to crt_rpc_complete/RPC_ERROR, but its not a big deal, we could look at stacktrace